internal/server/filesystem/server_test.go

```diff
@@ -0,0 +1,238 @@
+package filesystem
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal"
+)
+
+type fakeFileSystemAPI struct{}
+
+func (fakeFileSystemAPI) PathExists(path string) (bool, error) {
+	return true, nil
+}
+func (fakeFileSystemAPI) Mkdir(path string) error {
+	return nil
+}
+func (fakeFileSystemAPI) Rmdir(path string) error {
+	return nil
+}
+func (fakeFileSystemAPI) LinkPath(tgt string, src string) error {
+	return nil
+}
+
+func TestMkdirWindows(t *testing.T) {
+	v1alpha1, nil := apiversion.NewVersion("v1alpha1")
```

Comment:
Wasn't aware golang had some pattern matching ;)

This only compiles because you shadow `nil`, then use that on L114. Please remove?

Reply:
heh - can't believe that compiled. Will remove :-)